crates/pallets/starknet/src/types.rs

```diff
@@ -22,6 +24,8 @@ pub type StorageSlot = (StorageKey, Felt252Wrapper);
 pub type CasmClassHash = ClassHash;
 pub type SierraClassHash = ClassHash;
 
+pub type TransactionSimulationResult = Result<TransactionExecutionInfo, PlaceHolderErrorTypeForFailedStarknetExecution>;
```

Comment:
This is the sencond time you define this type.
Define it in primitive/simulations instead, so you can access it from both crates